lang/en/local_augmented_teacher.php

```diff
@@ -124,6 +124,16 @@
 $string['currentlyselectedusers'] = 'Currently selected users';
 $string['allfieldsrequired'] = 'All fields are required';
 $string['previewhtml'] = 'HTML format preview';
+/**
+ * $a: {
+ *      id: int,
+ *      fullname: string,
+ *      shortname: string,
+ *      url: string,
+ *      link: string (HTMLElement <a>...</a>)
+ * }
+ */
+$string['sendfromcourse'] = '<p style="text-align: right;">Send from: {$a->link}</p>';
```

Comment:
Se om du kan undgå HTML i sprogstrengene og i stedet for have det i Mustache skabelonen.

Reply:
Det er ikke vores plugin.
Vi kan ikke introducere en ny måde at udskrive ting med.
Også at hver site skal måske ha' forskellige ting i deres besked.